Sources/MapboxDirections/Intersection.swift

```diff
@@ -4,7 +4,10 @@ import Turf
 /**
  A single cross street along a step.
  */
-public struct Intersection {
+public struct Intersection: ForeignMemberContainer {
+    public var foreignMembers: JSONObject = [:]
+    public var lanesForeignMembers: [JSONObject] = []
```

Comment:
This falls our of usual pattern, but since `Intersection` takes care of Coding lanes data - there is no better place to store Lane's foreign members.